e2e-tests/.ci/server.generate.sh

```diff
@@ -298,8 +297,8 @@ generate_env_files() {
 	BUILD_ID
 	CI_BASE_URL
 	BROWSER
-	AUTOMATION_DASHBOARD_URL
```

Comment:
FYI tthese two lines were not needed, since we anyway loaded the optional automation dashboard's variables in `.env.cypress` in these lines: https://github.com/mattermost/mattermost/pull/25367/files#diff-ec7b68e669acf8577f2dc1d58738b34eec6155a62e3c1085fc7811bbecc76416L331-L334